client/my-sites/patterns/components/page-view-tracker.tsx

```diff
@@ -38,19 +46,19 @@ export function PatternsPageViewTracker( { category, searchTerm }: PatternsPageV
 	}, [ category, isDevAccount, isLoggedIn ] );
```

Comment:
I think we could wrangle the updated properties for the `calypso_pattern_library_filter` event too in this PR.